MaxText/configs/base.yml

```diff
@@ -52,6 +52,9 @@ metrics_file: "" # for testing, local file that stores scalar metrics. If empty,
 # If true save metrics such as loss and TFLOPS to GCS in {base_output_directory}/{run_name}/metrics/
 gcs_metrics: False
 
+# If true save train config to {base_output_directory}/{run_name}/
+gcs_train_config: False
```

Comment:
I don't understand this flag name? Don't we want to save decode there as well?

Reply:
Thanks for pointing this out, I changed the flag name and added the config uploading to `decode.py` as well.